src/pages/testPage/index.tsx

```diff
@@ -45,23 +61,37 @@ const TestPage = () => {
         </Stack>
         <StylizedPaper title='”Node.js” question'>
           <span className='mx-auto text-base mb-2.5 font-semibold text-xl text-center'>
-            What is the correct JavaScript syntax to change the content of the
-            HTML element below?
+            {questions[numberOfQuestion].description}
           </span>
-          <TestQuestions testQuestions={testQuestions} />
+          <TestQuestions
+            answers={questions[numberOfQuestion].content.options}
+          />
           <Stack
             direction='row'
             justifyContent='center'
             alignItems='center'
             spacing={4}
           >
             <Button color='info'>Skip</Button>
-            <Button onClick={nextQuestionHandler}>Next</Button>
+            <Button disabled={disabled} onClick={nextQuestionHandler}>
+              Next
+            </Button>
           </Stack>
         </StylizedPaper>
       </Stack>
     </Layout>
   );
-};
+}
+
+export const getServerSideProps = wrapper.getServerSideProps(
+  (store) => async () => {
+    const res = await questionsApi.getQuestions();
```

Comment:
api endpoint also should be called by redux

Reply:
i understood you. I will use createAsyncThunk